### PR TITLE
Fixes param_true? issue from PR #212 and ubuntu 12 static networking

### PR DESCRIPTION
**Pull request by dgagnon**
this one should be the good one.

### CODE REVIEW

File: kickstart/finish.erb [hunk 1]
```diff
@@ -34,7 +34,7 @@ echo 'root:<%= root_pass -%>' | /usr/sbin/chpasswd -e
 <% end -%>
 
 <% if puppet_enabled -%>
-<% if @host.param_true?('enable-puppetlabs-repo') -%>
+<% if @host.params['enable-puppetlabs-repo'] -%>
```
Comment (mmoll) [inline kickstart/finish.erb:37]: the problem here is that global(!) parameters in Foreman are strings, not booleans, so also their value has to be checked,  try for yourself :balloon::

``` ruby
foo = 'true'
bar = 'false'

if foo
  puts 'foo is true'
end

if bar
  puts 'bar is true'
end
```

**Comment by mmoll** [comment]
merged the static networking fix, thanks @dgagnon!

**Comment by lzap** [comment]
I still see test failures in this repo, can you take a look?

**Comment by lzap** [comment]
I have fixed those, don't worry.

**Comment by domcleal** [comment]
This PR didn't break any.  It was #212 and the failures were filed in #215, I filed #222.

**Pull request closed without merging**